core/src/components/badge/badge.common.scss

```diff
@@ -63,3 +63,20 @@
 :host(:not(.in-tab-button)[vertical].badge-vertical-bottom) {
   bottom: 0;
 }
+
+// Badge in Button
+// --------------------------------------------------
+
+:host(:not(:empty).in-button) {
```

Comment:
This CSS should be moved to badge.native.scss, along with the vars to badge.native.vars.scss.
Then this css should be repeated on ionic.theme.scss but using the design tokens directly. 

This way, wee keep the consistency with the scss architecture of other patterns, between design systems, and if in the future the tokens change values, we won't need to manually change this CSS for the Ionic Theme.